### DISCUSSION TITLE
How to import wox

### DISCUSSION THREAD

**Discussion by jhyan-ux** [Wox-launcher/Wox#4077] [Q&A]
I try to build my owned wox plugin, but I can not install wox in my environment with "pip install wox". Please help. Thanks!

**Comment by jhyan-ux** [comment]
wox version: [v1.4.1196](https://github.com/Wox-launcher/Wox/releases/tag/v1.4.1196)
using the full-installer

**Comment by CursedCheese** [comment]
It should work if you place the Wox package in the proper path without having to manually install it. In fact, when you open the plugin code in an editor, it will warn you that wox is not recognized, but it works fine.

The path to place the plugin should be `Appdata/Local/Wox/app-1.4.1196/Plugins`.

If this does not work, please let me know.
